interface/portraits/portraits_frostvrykul_sprites.gfx

```diff
@@ -529,41 +59,14 @@ spriteTypes = {
 	spriteType = {
 		name = "GFX_frostvrykul_female_hair"
```

Comment:
This sprite and image aren't needed? `vrykul_female_hair_1` and `frostvrykul_female_hair_1` are the same.
<details>
<summary>vrykul_female_hair_1</summary>

![image](https://user-images.githubusercontent.com/46384992/77200806-c0635d00-6b04-11ea-9e1c-e1612c32ee1f.png)

</details>
<details>
<summary>frostvrykul_female_hair_1</summary>

![image](https://user-images.githubusercontent.com/46384992/77200821-c6f1d480-6b04-11ea-86ab-d16bb4e99974.png)

</details>

Reply:
Undead females share the same hairs too.